lld/COFF/Driver.h

```diff
@@ -236,6 +236,9 @@ class LinkerDriver {
   void parseSection(StringRef);
   void parseAligncomm(StringRef);
 
+  // Parses a MS-DOS stub file
+  void parseStub(StringRef path);
```

Comment:
I think if you renamed this to `parseDosStub` there would be no need to have a comment for it.